.github/workflows/ci.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       fail-fast: false
       matrix:
         version:
-          - '1.6' # Replace this with the minimum Julia version that your package supports.
```

Comment:
While we're modifying CI config, could we bump the versions of actions and switch over to the more useful https://github.com/julia-actions/cache?